index.bs

```diff
@@ -298,29 +301,46 @@ class AV1MultiFrameSampleGroupEntry extends VisualSampleGroupEntry('av1M') {
 CMAF AV1 track format and CMAF media profiles {#cmaf}
 =====================================================
 
-When fragmented or segmented, for example for the purpose of adaptive streaming, tracks conforming to this specification SHALL also conform to the constraints given in section 9.2 of CMAF, and are called CMAF AV1 Tracks.
-When encrypted, CMAF AV1 Tracks SHALL use the signaling defined in CMAF, which in turns relies on CENC, with the provisions given in the “Common Encryption” Section below.
-A CMAF AV1 Track SHALL contain a valid AV1 bitstream with the following constraints:
-- The track may use multiple sample entries,
-- The following values SHALL not change in the track:
+[[CMAF]] defines structural constraints on ISOBMFF files additional to [[ISOBMFF]] for the purpose, for example of adaptive streaming or for encrypted files. [[CMAF]] also relies on the signaling of CMAF Media Profiles. This document specifies CMAF Media Profiles and associated brands for AV1 bitstreams in CMAF-compliant files.
```

Comment:
Change
  purpose, for example of adaptive 
To
  purpose of, for example, adaptive